break_glass/program.py

```diff
@@ -0,0 +1,119 @@
+"""This program orchestrates the request and approval process for break glass scenarios.
+
+Break glass scenarios occur when a developer needs to access sensitive data or perform
+a critical operation that requires elevated permissions beyond their usual access.
+
+Workflow:
+    1. A developer initiates the process by using a Slack slash command to request
+       break glass approval.
+    2. AutoKitteh sends a form to the developer, requesting details about the reason
+       for the elevated access.
+    3. The developer fills out and submits the form, providing the necessary information
+       and justification for the request.
+    4. AutoKitteh sends a notification to the SRE (Site Reliability Engineering) team
+       with an approve/deny message, including the details of the request.
+    5. The SRE team reviews the request and makes a decision to approve or deny the request.
+    6. AutoKitteh sends a message to the developer with the decision, notifying them
+       whether the request was approved or denied.
+
+The program integrates with Jira to verify ticket existence and ensure that the requester
+is the assignee of the ticket. It also uses Slack for communication and notifications
+throughout the process.
+"""
+
+import os
+from pathlib import Path
+
+import autokitteh
+from autokitteh.atlassian import atlassian_jira_client
+from autokitteh.slack import slack_client
+from requests.exceptions import HTTPError
+
+
+APPROVER = "sre-team"
```

Comment:
We want this to be configurable. Keep this global value, but initialize it with `os.getenv` (i.e. read a project var)

I'd also rename this to `APPROVAL_CHANNEL` - but only if it doesn't mess up the code by being longer.